cli/server/server.go

```diff
@@ -491,7 +491,7 @@ func startServer(ctx *cli.Context) error {
 
 	go serv.Start(errChan)
 	if !cfg.ApplicationConfiguration.RPC.StartWhenSynchronized {
-		rpcServer.Start()
+		go rpcServer.Start()
```

Comment:
We have https://github.com/nspcc-dev/neo-go/issues/2896 for it.

Reply:
tagged